libraries/RainMaker/examples/RMakerCustom/RMakerCustom.ino

```diff
@@ -130,11 +133,13 @@ void loop()
           RMakerWiFiReset(2);
         } else {
           // Toggle device state
-          dimmer_state = !dimmer_state;
-          Serial.printf("Toggle State to %s.\n", dimmer_state ? "true" : "false");
-          my_device.updateAndReportParam(ESP_RMAKER_DEF_POWER_NAME, dimmer_state);
-          (dimmer_state == false) ? digitalWrite(gpio_dimmer, LOW) : digitalWrite(gpio_dimmer, HIGH);
-      }
+            dimmer_state = !dimmer_state;
+            Serial.printf("Toggle State to %s.\n", dimmer_state ? "true" : "false");
+            if (my_device) {
+                my_device->updateAndReportParam(ESP_RMAKER_DEF_POWER_NAME, dimmer_state);
+            }
+            (dimmer_state == false) ? digitalWrite(gpio_dimmer, LOW) : digitalWrite(gpio_dimmer, HIGH);
+        }
```

Comment:
Please follow same styling across the file

Reply:
We are losing time with style issues when an automatic script with clang formatter could keep all PR at same pattern.

Reply:
We have a script in IDF which uses `astyle` programme to achieve this: [here](https://github.com/espressif/esp-idf/blob/master/tools/format.sh). However this has to be run manually for this repo.

If you have a change for using automated clang formatter, would you please raise a PR for the same?

Reply:
I'm not an expert in this subject otherwise I could make a PR. Actually I'm using uncrustify in VSCode only.

Anyway there's an action with example for artistic style here:
 https://github.com/marketplace/actions/artistic-style